src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -370,7 +357,7 @@ else if (ch == ':')
                         }
                         // same state
                     } else {
-                        builder = new StringBuilder();
+                        StringBuilder builder = new StringBuilder();
```

Comment:
Why are these else cases handled differently? I think these should also be replaced by `result.append(ch);`?

Reply:
Because this appends a name and not result. This method has fall through logic that forces us to retain some original logic.